spec/lib/submission_builder/ty2024/states/nj/documents/nj2450_spec.rb

```diff
@@ -24,7 +25,10 @@
         body_elements.each do |body|
           w2 = [w2_1, w2_2].find { |test_w2| test_w2.employer_ein == body.at('FedEmployerId').text}
 
-          expect(body.at('EmployerName').text).to eq(w2.employer_name)
+          expect(body.at('EmployerName').text).to eq(w2.employer_name[0...35])
+          if w2.employer_name == long_employer_name
+            expect(body.at('EmployerName').text.length).to eq(35)
```

Comment:
[pebble] putting expectations inside an `if` feels off to me, since there's an outside chance the test is only passing because the expectation isn't being tested. I think the expectation on line 28 is probably enough on its own, but if you feel there needs to be more then you could test explicitly the two expected names outside of the `.each`?